docs/opentracing.asciidoc

```diff
@@ -6,6 +6,8 @@ endif::[]
 [[opentracing-bridge]]
 == Elastic APM OpenTracing bridge
 
+NOTE: Latest supported OpenTracing version: 0.33 (as of agent and OpenTracing-bridge version 1.9.0)
```

Comment:
0.31.0 and 0.32.0 are still supported, right?

Reply:
Yes, does this imply it is not? How would you put it?